client/src/modules/dashboard/index.jsx

```diff
@@ -91,11 +92,13 @@ class Dashboard extends Component {
       pendingForListCreation,
       pendingForLists
     } = this.state;
+    const breadcrumbs = [{ name: 'dashboard', path: '/dashboard' }];
 
     return (
       <Fragment>
         <Toolbar />
```

Comment:
I think that the toolbar should be moved to the Layout component now. All components render the toolbar but don't use it in any way.